pkg/util/fixtures/populate/populate_test.go

```diff
@@ -0,0 +1,56 @@
+package populate
+
+import (
+	"crypto/rsa"
+	"crypto/x509"
+	"fmt"
+	"strconv"
+	"testing"
+
+	. "github.com/onsi/gomega"
+)
+
+func TestPopulate(t *testing.T) {
```

Comment:
all of this can go away - replace it with tests which walk `api.OpenShiftManagedCluster` (for both the internal and external api), yaml marshal the result and compare it against a known good value.